README.md

```diff
@@ -1,75 +1,77 @@
+| Parameter | Type | Required? | Description |
 | --- | --- | --- | --- |
-| *cm* | String | Yes | An instance of ConnectionManager library v3.1.0 or above. |
-| *wakeReasonCallbacks* | Table | No | A table of optional wake reasons callbacks. See [below](#wake-reason-callbacks). Default: an empty table. |
-| *isDebug* | Boolean | No | Controls the debug output of the library. |
+| *cm* | Object | Yes | An instance of [ConnectionManager](https://github.com/electricimp/ConnectionManager) 3.1.0 or above |
+| *wakeReasonCallbacks* | Table | No | A table of optional [wake reason callbacks](#wake-reason-callbacks). Default: an empty table |
+| *isDebug* | Boolean | No | Controls the debug output of the library. Set to `true` for extra output. Default: `false` |
 
 #### Wake Reason Callbacks ####
 
-All keys are optional and values are callback functions.
+LPDeviceManager allows you to set code to be executed when the device wakes for a specific reason.
+
+All of the following keys are optional. Their values are callback functions that will be triggered in the event of the wake reasons for which the keys are named.
 
-| Key | Value | 
+| Key | Value |
 | --- | --- |
-| *onColdBoot* | Callback to be executed on the cold boot, the callback takes no parameters | 
-| *onSwReset* | Callback to be executed on a software reset, the callback takes no parameters |
-| *onTimer* | Callback to be executed after "deep" sleep timer expires, the callback takes no parameters |
-| *onInterrupt* | Callback to be triggered on a wakeup pin, the callback takes no parameters |
-| *onPowerRestored* | Callback to be triggered when the imp's VBAT is powered during a cold start, the callback takes no parameters. Note: this wakereason is only available on impOS v40 and above |
-| *defaultOnWake* | Callback that catches all other wake reasons, takes wake reason constant as a parameter [see the dev center for wake reason constant descriptions](https://developer.electricimp.com/api/hardware/wakereason) |
+| *onColdBoot* | Callback to be executed on a cold boot. The callback takes no parameters |
+| *onSwReset* | Callback to be executed on a software reset. The callback takes no parameters |
+| *onTimer* | Callback to be executed after a deep sleep timer fires. The callback takes no parameters |
+| *onInterrupt* | Callback to be triggered when a wakeup pin is asserted. The callback takes no parameters |
+| *onPowerRestored* | Callback to be triggered when the imp's V<sub>BAT</sub> is powered during a cold start. The callback takes no parameters. **Note** This wake reason is only available on impOS™ 40 and above |
+| *defaultOnWake* | Callback that catches [all other wake reasons](https://developer.electricimp.com/api/hardware/wakereason), and takes the impOS wake reason constant as its argument |
 
 #### Example ####
 
 ```squirrel
 // Wake up on timer flow
 function onScheduledWake() {
+    server.log("Device woke according to schedule");
```

Comment:
FYI - With connection manager defaults the device is not connected here, so this log will not show.